image/templates/helm/stackrox-secured-cluster/internal/cluster-config.yaml.tpl

```
@@ -29,5 +29,26 @@ clusterConfig:
       disableBypass: {{ ._rox.admissionControl.dynamic.disableBypass }}
       enforceOnUpdates: {{ ._rox.admissionControl.dynamic.enforceOnUpdates }}
     registryOverride: {{ ._rox.registryOverride }}
+    collectorConfig:
+      process_config:
```

The configuration must be camelCase to align with the config semantics of the file. The other configurations should be in camelCase too,

```suggestion
      processConfig:
```